absearch/tests/test_check.py

```diff
@@ -35,7 +35,7 @@ def test_check_fails():
 
 
 def test_check_enum_fails():
-    conf = os.path.join(os.path.dirname(__file__), '..', '..', 'data',
+    conf = os.path.join(os.path.dirname(__file__), '..', '..', 'example-data',
                         'config.json')
     os.rename(conf, conf + '.saved')
```

Comment:
I don't know if it is the right PR (or time) to address that, but I find the existing tests extremely ugly. The tests shouldn't mess up with example config files (rename them etc.). At worse there should be a `tests/data/` folder or something...

Reply:
agreed, will final an issue for this